src/coreclr/jit/lsra.cpp

```diff
@@ -8929,8 +8975,18 @@ void LinearScan::dumpLsraStatsSummary(FILE* file)
 }
 #endif // TRACK_LSRA_STATS
 
+#undef RBM_ALLFLOAT_USE
+#undef RBM_FLT_CALLEE_TRASH_USE
+#undef CNT_CALLEE_TRASH_FLOAT_USE
+
 #ifdef DEBUG
-void dumpRegMask(regMaskTP regs)
+// Please see the comment for these instance variables in `compiler.h`
+#if defined(TARGET_AMD64)
+#define RBM_ALLFLOAT_USE (compiler->rbmAllFloat)
```

Comment:
why can't you use the macros previously defined and undefed few lines back?

Reply:
I should leave a comment here: this `compiler` must match the parameter in `dumpRegMask`. I thought it more error prone to assume the name passed in here is the same one at the class level for the file.